ui/src/Components/ProfilePage.jsx

```diff
@@ -112,39 +112,78 @@ export default class ProfilePage extends React.Component {
           {!this.state.profile && this.state.loading == false ? (
             <div> Profile Does Not Exist </div>
           ) : (
-            <div className="row-buffer side-buffer left-align">
-              <h4>{this.state.profile.result.first_name} {' '}
-              {this.state.profile.result.last_name}</h4>
-              {this.state.profile.result.city ? <H4>Servicing the {this.state.profile.result.city} area</H4> : <div/>}
-              <Table className="row-buffer" responsive>
+            <div className="row-buffer side-buffer">
+               {this.state.profile.result.app_status == 'Began Application' ? (
+                  <Alert className="width-fit" color="primary">
+                    Your application is in draft mode.
```

Comment:
Thoughts on making this a global variable - usually const are much easily manipulated at the top of component set up rather in-line... thoughts?

Reply:
This is not a blocking concern, but more maintainability

Reply:
Was this comment looked at?